container-images/mds-attachment-service/package.json

```
@@ -8,8 +8,11 @@
     "dist/"
   ],
   "scripts": {
-    "bundle": "webpack --mode=production --env npm_package_name=${npm_package_name} --env npm_package_version=${npm_package_version}",
-    "image": "yarn bundle && ../../bin/build-helper buildImage mds-attachment-service:${npm_package_version}",
+    "bundle:development": "webpack --mode=development --env npm_package_name=${npm_package_name} --env npm_package_version=${npm_package_version}",
+    "bundle:production": "webpack --mode=production --env npm_package_name=${npm_package_name} --env npm_package_version=${npm_package_version}",
+    "image:development": "yarn bundle:development && ../../bin/build-helper buildImage mds-attachment-service:${npm_package_version}",
+    "image:production": "yarn bundle:production && ../../bin/build-helper buildImage mds-attachment-service:${npm_package_version}",
+    "image": "yarn image:production",
```

Knowing that `production` is the default mode, it may be possible to simplify these to something like:
```suggestion
    "bundle": "webpack --env npm_package_name=${npm_package_name} --env npm_package_version=${npm_package_version}"
    "bundle:development": "yarn bundle --mode=development",
    "image:development": "yarn bundle:development && ../../bin/build-helper buildImage mds-attachment-service:${npm_package_version}",
    "image": "yarn bundle && ../../bin/build-helper buildImage mds-attachment-service:${npm_package_version}",
```

---

Mmm yeah, I suppose. I do like leaning towards the explicit rather than implicit config, though (we had `--mode production` specified manually previously, so this just extends upon that). Anyone else have thoughts?